pkg/agentfs/examples/node.Dockerfile

```diff
@@ -40,13 +36,15 @@ RUN pnpm install --frozen-lockfile
 COPY . .
 
 # Build the project
-RUN pnpm run build
+RUN pnpm exec tsc
```

Comment:
what's the reason behind this change?

Reply:
I'm switching this to assume the existence of build, download-files, and start scripts and just ensuring the quickstart sets those up and we have docs about it